contracts/UpgradeabilityProxy.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity ^0.4.24;
```

Comment:
why this change?

Reply:
Hmmm... That's not mine. Probably linter?